service/matching/matchingEngine_test.go

```diff
@@ -194,11 +194,23 @@ func (s *matchingEngineSuite) TestAckManager() {
 }
 
 func (s *matchingEngineSuite) TestPollForActivityTasksEmptyResult() {
-	s.PollForTasksEmptyResultTest(persistence.TaskListTypeActivity)
+	s.PollForTasksEmptyResultTest(s.callContext, persistence.TaskListTypeActivity)
 }
 
 func (s *matchingEngineSuite) TestPollForDecisionTasksEmptyResult() {
-	s.PollForTasksEmptyResultTest(persistence.TaskListTypeDecision)
+	s.PollForTasksEmptyResultTest(s.callContext, persistence.TaskListTypeDecision)
+}
+
+func (s *matchingEngineSuite) TestPollForActivityTasksEmptyResultWithShortContext() {
+	callContext, cancel := context.WithTimeout(s.callContext, returnEmptyTaskTimeBudget+10*time.Millisecond)
```

Comment:
Lets put `returnEmptyTaskTimeBudget+10*time.Millisecond` into some meaning variable name. Its hard to just read that and know what is being tested.